ExRam.Gremlinq.Core/Models/IGraphModel.cs

```diff
@@ -8,5 +9,9 @@ public interface IGraphModel
 
         IGraphElementModel VerticesModel { get; }
         IGraphElementModel EdgesModel { get; }
+
+        object GetIdentifier(Expression expression);
+
+        object GetIdentifier(Type elementType, string memberName);
```

Comment:
Looks very good. I'd rather not have two ways of getting an identifier here though, the former should do.

Reply:
Ok, I removed the 2nd method and updated any references. Now, only `object GetIdentifier(Expression expression)` is exposed.